test/service/test_response.py

```diff
@@ -1447,7 +1450,13 @@ def test_pagination_search_after_search_before(self):
         ]
         self.assertEqual(expected_entry_ids, [h['entryId'] for h in response_json['hits']])
 
-        self.assertEqual(response_json['pagination']['search_after'], '"~null"')
+        # Note that the sort field 'workflow' is an analysis_protocol field and
+        # does not exist in all bundles. This is why the 'search_after' field
+        # has the value 'null' (json representation of None) because the last
+        # row in this page of results does not have an analysis_protocol or
+        # 'workflow' field. If the last row did have a 'workflow' field with
+        # value of None, 'search_after' would be a translated None ('"~null"')
```

Comment:
```suggestion
        # value of None, 'search_after' would be a translated None (`"~null"`)
```

ditto for the other literals in this comment.